### PR TITLE
README.adoc: Rewrite it before archiving

### PR DESCRIPTION
**Pull request by lsalvadore**
In particular, add a link to the main webpage for status reports on the FreeBSD website, so that visitors of the repository know where to check for up to date information.

### CODE REVIEW

**Comment by debdrup** [comment]
I threw a few ideas into the metaphorical pot in a response to the email you sent, but I think you're right that we're approaching the point at which we can do the needful.

File: README.adoc [hunk 1]
```diff
@@ -1,71 +1,6 @@
...
-Collated reports are:
-
-* Sent to link:https://lists.freebsd.org/subscription/freebsd-announce[freebsd-announce]. 
-* Listed at link:https://www.freebsd.org/news/status/[FreeBSD Status Reports].
-* Promoted through link:https://www.freebsd.org/news/newsflash/[FreeBSD News Flash].
-
-The most recent report:
-
-* https://www.freebsd.org/status/report-2022-07-2022-09/
+Please check https://www.FreeBSD.org/status/ for up to date information about FreeBSD status reports collection, reviewing and publication.
```
Comment (pauamma) [inline README.adoc:6]: I'd make the URL a link, perhaps `Please check for link:https://www.FreeBSD.org/status/[up to date information about FreeBSD status reports collection, reviewing and publication].`

Comment (lsalvadore) [reply README.adoc:6]: Good idea. How about `Please check the link:https://www.FreeBSD.org/status/[FreeBSD status reports webpage] for up to date information about status reports collection, reviewing and publication.`?

Comment (pauamma) [reply README.adoc:6]: > Good idea. How about `Please check the link:https://www.FreeBSD.org/status/[FreeBSD status reports webpage] for up to date information about status reports collection, reviewing and publication.`?

Even better.

**Review by pauamma — APPROVED** [review]

File: README.adoc [hunk 2]
```diff
@@ -1,71 +1,6 @@
 = FreeBSD Quarterly Status Reports
 
-== Drafts
+This repository has been used to collect and review FreeBSD quarterly status reports from 2017Q4 through 2022Q4.
```
Comment (grahamperrin) [inline README.adoc:3]: > has been

instead: 

> was

Comment (grahamperrin) [inline README.adoc:3]: > FreeBSD quarterly status reports

instead: 

> reports

(No need to repeat what's in the heading that precedes the sentence).

File: README.adoc [hunk 3]
```diff
@@ -1,71 +1,6 @@
...
-Collated reports are:
-
-* Sent to link:https://lists.freebsd.org/subscription/freebsd-announce[freebsd-announce]. 
-* Listed at link:https://www.freebsd.org/news/status/[FreeBSD Status Reports].
-* Promoted through link:https://www.freebsd.org/news/newsflash/[FreeBSD News Flash].
-
-The most recent report:
-
-* https://www.freebsd.org/status/report-2022-07-2022-09/
+Please check the link:https://www.FreeBSD.org/status/[FreeBSD status reports webpage] for up to date information about FreeBSD status reports collection, reviewing and publication.
```
Comment (grahamperrin) [inline README.adoc:6]: > information about FreeBSD status reports collection, reviewing and publication.

instead: 

> information.

**Review by grahamperrin — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 2 above)
Comment (lsalvadore) [reply README.adoc:3]: Right, I am going to change it.

Comment (lsalvadore) [reply README.adoc:3]: I prefer to keep it here: stressing that the reports were being published on a quarterly basis makes it clearer what 2017Q4 and 2022Q4 mean. I do not know if in the English speaking world such notations are vey common, but in Italian and French I have seen them very rarely: indeed, in those two languages I usually see those kinds of periods refered as trimesters. So I think repeating "quarter"  might help non native English speaker understand better what we are talking about.

File: README.adoc (hunk 3 above)
Comment (lsalvadore) [reply README.adoc:6]: I prefer to keep the long form. Information is vague and could refer to anything, including statistics, lists of reports submitters and other kinds of things that we actually do not have. On the other hand, we have and should still have in the future information about collection, reviewing and publication.

**Pull request merged**